bears/scss/SCSSLintBear.py

```diff
@@ -21,4 +23,195 @@ class SCSSLintBear:
+                        allow_trailing_semicolon: bool=True):
+        '''
+        :param check_bang_format:
+            Reports when you use improper spacing around ``!`` (the "bang") in
+            ``!default``, ``!global``, ``!important``, and ``!optional`` flags.
+        :Param space_before_bang:
+            Enforces a space before ``!`` (the "bang").
+        :param sapce_after_bang:
+            Enforces a space after ``!`` (the "bang").
+        :param allow_chained_classes:
+            Allows defining a rule set using a selector with chained classes.
+        :param allow_color_keywords:
+            Prefers color keywords over hexadecimal color codes.
+        :param use_color_variables:
+            Prefers color literals (keywords or hexadecimal codes) to be used
+            only in variable declarations.
+        :param allow_debug_statement:
+            Allows ``@debug`` statements.
+        :param check_declaration_order:
+            Rule sets should be ordered as follows: `@extend` declarations,
+            `@include` declarations without inner `@content`, properties,
+            `@include` declarations with inner `@content`, then nested rule
+            sets.
+        :param allow_duplicate_properties:
+            Allows defining the same property twice in a single rule set.
+        :param allow_consecutives_duplicate_property:
+            Allows defining the same property consecutively in a single rule
+            set.
+        :param else_placement_style:
+            Places`@else` statements on the same line as the preceding curly
+            brace.
+        :param force_empty_line_between_blocks:
+            Separate rule, function, and mixin declarations with empty lines.
+        :param allow_empty_rules:
+            Allows empty rule set.
+        :param hexadecimal_length_style:
+            Prefer shorthand or long-form hexadecimal colors by setting the
+            style option to short or long, respectively.
+        :param hexadecimal_notation_case:
+            Checks if hexadecimal colors are written in lowercase or uppercase.
+        :param allow_hexadecimal_validation:
+            Ensure hexadecimal colors are valid (either three or six digits).
+        :param allow_id_selector:
+            Allows using ID selectors.
+        :param allow_important_rule_in_property:
+            Allows using !important in properties.
+        :param use_spaces:
+            Use spaces for indentation (tabs otherwise).
+        :param indent_size:
+            Number of spaces per indentation level.
+        :param leading_zero:
+            Determines whether leading zeros should be written or not in
+            numeric values with a decimal point.
+        :param allow_mergeable_selectors:
+            Allows defining the same selector twice in a single sheet.
+        :param allow_leading_underscore:
+            Allows names to start with a single underscore.
+        :param function_name_case:
```

Comment:
for the FilenameBear we use `file_naming_convention` apparently, you should ideally stick to that scheme and make sure it's good across all bears. I think at least one other bear might have a setting like this. The options we offer for filenames are `snake`, `camel`, `pascal`, see https://github.com/coala-analyzer/bear-docs/blob/master/docs/FilenameBear.rst